sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -751,14 +558,14 @@ export class CertificateClient {
         }
 
         if (updatedOptions.enabled !== undefined) {
-          generatedOptions.attributes = {
+          params.attributes = {
             enabled: updatedOptions.enabled,
           };
         }
 
         let parsedBody: any;
-        await this.client.setCertificateIssuer(this.vaultUrl, issuerName, provider, {
-          ...generatedOptions,
+        await this.client.setCertificateIssuer(issuerName, params, {
```

Comment:
Shape here has changed, I believe since we don't flatten anymore. This affects most operations